free-programming-books-fr.md

```diff
@@ -254,6 +255,11 @@
 * [Tutoriel Ruby on Rails : Apprendre Rails par l'exemple](http://french.railstutorial.org/chapters/beginning) - Michael Hartl
 
 
+### Rust
+* [Traduction du Rust book en français](https://jimskapt.github.io/rust-book-fr/) par Thomas Ramirez
```

Comment:
```suggestion
### Rust

* [Traduction du Rust book en français](https://jimskapt.github.io/rust-book-fr/) par Thomas Ramirez
```

Hello, the error occurs from the Travis-CI Build for the reason "Incorrect number of blank lines between heading and section."

If you refer to the [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) file, it mentions how many spaces should be between the heading and the resources.

In your case you do have "2 empty lines between last link and new section." However, you do not have "1 empty line between heading & first link of its section." Simply fixing this should pass the Travis-CI build!